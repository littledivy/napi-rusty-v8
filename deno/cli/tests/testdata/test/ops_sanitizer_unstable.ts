Deno.test("no-op", function () {});
Deno.test("leak interval", function () {
  setInterval(function () {});
});
