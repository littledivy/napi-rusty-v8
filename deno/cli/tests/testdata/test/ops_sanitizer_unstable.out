Check [WILDCARD]/testdata/test/ops_sanitizer_unstable.ts
running 2 tests from [WILDCARD]/testdata/test/ops_sanitizer_unstable.ts
test no-op ... ok ([WILDCARD])
test leak interval ... FAILED ([WILDCARD])

failures:

leak interval
AssertionError: Test case is leaking async ops.
Before:
  - dispatched: 1
  - completed: 1
After:
  - dispatched: [WILDCARD]
  - completed: [WILDCARD]
Ops:
  op_sleep:
    Before:
      - dispatched: 1
      - completed: 1
    After:
      - dispatched: [WILDCARD]
      - completed: [WILDCARD]

Make sure to await all promises returned from Deno APIs before
finishing test case.
    at [WILDCARD]

failures:

	leak interval

test result: FAILED. 1 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out ([WILDCARD])

error: Test failed
